### PR TITLE
Updating VR polyfill to 0.10.4

### LINKED ISSUE #113: upgrade version of `webvr-polyfill` dependency
I'm filing this to separate the issue of upgrading `webvr-polyfill` from fixing the source-map JS error raised in issue #54.

<hr>

From Slack:

> **caseyyee [10:48 PM]**
> @cvan ok, looks like with the polyfill, all the desktop mouse stuff was removed and they just use orbit controls on desktop.  it doesn’t even return any polyfilled vrdisplays
> so it's basically strictly for mobile only

> **cvan [10:06 PM]**
> the new one has updates for the Sensor API
> **cvan [10:06 PM]**
> devicemotion is being deprecated
> **cvan [10:06 PM]**
> and it fixes a few 1.1 compliance issues

> **caseyyee [10:54 PM]**
> Yeah, that was my thought.   Provide some kind of sensible desktop usage.  maybe some FPS controls.
> is there any major reason to use the new polyfill?   I can update the polyfill and add back some default controls as a update.
> so do that after we push to the asset store.

### PR DESCRIPTION
**Pull request by delapuente**
Fix #113 

Orbital controls are missing since they were dropped in 0.10.0. Will fix them in a different bug.

### CODE REVIEW

File: Assets/WebGLTemplates/WebVR/index.html [hunk 1]
```diff
@@ -77,7 +77,8 @@ <h3>You&rsquo;ll need a <a href="https://webvr.rocks/">WebVR-enabled browser</a>
     </div>
 
     <script src="gl-matrix-min.js"></script>
-    <script src="webvr-polyfill.min.js"></script>
+    <script src="webvr-polyfill.min.new.js"></script>
+    <script>var vrPolyfill = new WebVRPolyfill();</script>
```
Comment (cvan) [inline Assets/WebGLTemplates/WebVR/index.html]: can we handle this in `webvr.js` or are there issues with doing that?

File: Assets/WebGLTemplates/WebVR/index.html [hunk 2]
```diff
@@ -77,7 +77,8 @@ <h3>You&rsquo;ll need a <a href="https://webvr.rocks/">WebVR-enabled browser</a>
     </div>
 
     <script src="gl-matrix-min.js"></script>
-    <script src="webvr-polyfill.min.js"></script>
+    <script src="webvr-polyfill.min.new.js"></script>
```
Comment (cvan) [inline Assets/WebGLTemplates/WebVR/index.html]: can we remove the old one? assuming we get in FPS controls (#125) by the next release/tag, it should be fine to remove the old version? is that fine?

File: Assets/WebGLTemplates/WebVR/webvr.js [hunk 3]
```diff
@@ -314,10 +314,7 @@
 
   // Check to see if we are using polyfill.
   function isPolyfilled(display) {
-    return (display.deviceId || '').indexOf('polyfill') > 0 ||
-        (display.displayName || '').indexOf('polyfill') > 0 ||
-        (display.deviceName || '').indexOf('polyfill') > 0 ||
-        display.hardwareUnitId;
+    return display.isPolyfilled;
```
Comment (cvan) [inline Assets/WebGLTemplates/WebVR/webvr.js]: :+1: glad you caught that; it was on my mental todo list

**Comment by delapuente** [comment]
To be honest, I never got it working on my OnePlus 3T. I'm debugging with the device emulator of Chrome.

**Pull request merged**